caravel/assets/javascripts/reduxUtils.js

```diff
@@ -74,3 +74,22 @@ export function enhancer() {
   }
   return enhancerWithPersistState;
 }
+
+export function areArrayDifferent(arr1, arr2) {
```

Comment:
I went with different because it matches `shouldComponentUpdate`'s bool. I can flip it.